x-pack/plugins/security_solution/server/usage/collector.ts

```diff
@@ -65,6 +66,163 @@ export const registerCollector: RegisterCollector = ({
         },
       },
       detectionMetrics: {
+        detection_rules: {
+          detection_rule_usage: {
+            query: {
+              enabled: { type: 'long', _meta: { description: 'Number of query rules enabled' } },
```

Comment:
Suggestion: Generally we use `enabled` to indicate if a feature is enabled/disabled by the users. It might be more verbose to name this `enabled_count` or something along

Reply:
Thanks, @Bamieh! There is a new team member in Data Engineering. I will send them in to work on the nits!
I'll put this on the task list